src/Nethermind/Nethermind.Consensus/Processing/CensorshipDetector/CensorshipDetectorConfig.cs

```diff
@@ -0,0 +1,9 @@
+// SPDX-FileCopyrightText: 2024 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+namespace Nethermind.Consensus.Processing.CensorshipDetector;
+
+public class CensorshipDetectorConfig : ICensorshipDetectorConfig
+{
+    public string[]? AddressesForCensorshipDetection { get; set; } = null;
```

Comment:
I would consider adding a boolean flag, Enabled. If Enabled is true, we will not listen to events, etc. For new heuristics, we might use something other than addresses in the config.

If Enabled is false, I would expect that we would have a NoCensorshipDetection class and do nothing. The default setting should be false

Reply:
If disabled, don't create `CensorhipDetector` at all